src/lib/openapi-to-json-schema.js

```diff
@@ -168,23 +168,27 @@ function searchForExampleByPointer(pointer, examples = []) {
  * @link https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.3.md
  * @param {Object} data
  * @param {Object} opts
- * @param {Object[]} opts.prevSchemas - Array of parent schemas to utilize when attempting to path together examples.
  * @param {String} opts.currentLocation - Current location within the schema -- this is a JSON pointer.
  * @param {Object} opts.globalDefaults - Object containing a global set of defaults that we should apply to schemas that match it.
  * @param {Boolean} opts.isPolymorphicAllOfChild - Is this schema the child of a polymorphic `allOf` schema?
+ * @param {Object[]} opts.prevSchemas - Array of parent schemas to utilize when attempting to path together examples.
+ * @param {Function} opts.refLogger - A function that's called anytime a (circular) `$ref` is found.
```

Comment:
Is `opts.refLogger` supposed to be a configurable hook?

Reply:
Yeah we supply that option within in `get-parameters-as-json-schema` so we can optionally add `components` into each schema group that needs it.

Reply:
Do you think this needs to be documented further?

Reply:
I think it's ok